python_modules/dagster/dagster_tests/daemon_tests/test_dagster_daemon.py

```diff
@@ -78,3 +78,12 @@ def test_daemon_json_logs(
 
         assert lines
         assert [json.loads(line) for line in lines]
+
+
+def test_daemon_rich_logs() -> None:
+    with instance_for_test() as instance:
+        daemon_controller_from_instance(
+            instance,
+            workspace_load_target=EmptyWorkspaceTarget(),
+            log_format="rich",
+        )
```

Comment:
seems like this test got cut off? Or is the idea that it's hard to test formatting in the rich case. If the latter, a comment around what the test is actually trying to accomplish would be nice.

Reply:
Yeah, we're just threading through the value to see that there's no error. Doesn't really make sense to test the format like JSON logging.